test/tests/DatePickerInput-test.js

```diff
@@ -106,4 +106,34 @@ describe('DatePickerInput', function() {
   });
 
 
+  it('DatePicker should work with valueLink', function() {
```

Comment:
as discussed offline, this test is not testing what it claims.
Please review it